tabulator/writers/json.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import six
+import json
+
+# from tabulator import Stream
+
+from ..writer import Writer
+from .. import helpers
+
+
+# Module API
+
+class JSONWriter(Writer):
+    """JSON writer.
+    """
+
+    # Public
+
+    options = [
+        'delimiter',
```

Comment:
Please have the `keyed` option here instead of `delimiter'

By default, I think it should save data as here - https://github.com/frictionlessdata/tabulator-py/blob/master/data/table-lists.json

And if `keyed=True` as here - https://github.com/frictionlessdata/tabulator-py/blob/master/data/table-dicts.json (what you've implemented already)